src/defaults/html.pipe.js

```diff
@@ -31,8 +33,10 @@ const htmlpipe = (cont, payload, action) => {
   action.logger.log('debug', 'Constructing HTML Pipeline');
   const pipe = new Pipeline(action);
   pipe
+    .tap(dump).when(() => !production())
```

Comment:
we have `pre`, `once`, `post`, why not call it `each` or `every` ?

Reply:
`tap` was the name suggested in https://www.amazon.com/dp/B0777JHD5M